src/main/scala/org/ergoplatform/nodeView/mempool/ErgoMemPool.scala

```diff
@@ -39,6 +41,25 @@ class ErgoMemPool private[mempool](pool: OrderedTxPool, private[mempool] val sta
 
   override def take(limit: Int): Iterable[ErgoTransaction] = pool.orderedTransactions.values.take(limit)
 
+  def random(limit: Int): Iterable[ErgoTransaction] = {
+    val result = mutable.ArrayBuffer[ErgoTransaction]()
+    val txSeq = pool.orderedTransactions.values.toSeq
```

Comment:
`TreeMap(1 -> 1).values.toSeq` produces `Stream` ... this is better `TreeMap(1 -> 1).values.to[Vector]`

Reply:
done in https://github.com/ergoplatform/ergo/pull/1570/commits/d7f185b2dee59d75b4cf0abd79a346afabf50276